src/types/global.d.ts

```diff
@@ -9,6 +9,9 @@ interface restrictionsProp {
     setVegan: React.Dispatch<React.SetStateAction<boolean>>;
     setVegetarian: React.Dispatch<React.SetStateAction<boolean>>;
 }
+interface guestProp {
```

Comment:
Choice one! For this guestProp or guestList, only one is needed.

Reply:
i fix  i fix